### PR TITLE
Overhauled how linebreaks works for moodlets.

### PR DESCRIPTION
**Pull request by PestoVerde322**
## About The Pull Request

This PR ~~just fixes the Syndicate balloon's moodlet not linebreaking and drammatically break the immersion.~~ Overhauls how moodlets linebreak works, fixes the syndicate balloon's moodlet not linebreaking properly, alongside some spare changes.

## Why It's Good For The Game

Fix this from happening:

dreamseeker_rKCdSFNcDt

Fixed:

image

## Changelog
:cl:
code: Overhauled the moodlet code to linebreak whenever a new moodlet is added
code: Moved the pool moodlet into their own dedicated files.
fix: Fixed the moodlet on the Syndicate Balloon
del: removed and reworked an overused meme moodlet description
/:cl:

### CODE REVIEW

File: code/datums/mood_events/generic_positive_events.dm [hunk 1]
```diff
@@ -85,7 +85,7 @@
 	hidden = TRUE
 
 /datum/mood_event/badass_antag
-	description = "I'm a fucking badass and everyone around me knows it. Just look at them; they're all fucking shaking at the mere thought of me around."
+	description = "I'm a fucking badass and everyone around me knows it. Just look at them; they're all fucking shaking at the mere thought of me around.\n"
```
Comment (itsmeow) [inline code/datums/mood_events/generic_positive_events.dm]: What dumbass coded it this way. Yeet this nonsense convention

**Review by itsmeow — DISMISSED** [review]

**Review by tontyGH — COMMENTED** [review]
This works, but you could replicate how `mob/examinate()` does it and make `msg` a list to store all the messages, and then do `jointext(msg, "\n")` so you can get rid of every `\n` in `datum/component/mood.dm` as well

**Comment by PestoVerde322** [comment]
> This works, but you could replicate how `mob/examinate()` does it and make `msg` a list to store all the messages, and then do `jointext(msg, "\n")` so you can get rid of every `\n` in `datum/component/mood.dm` as well

I think the whole examinate call is totally different than what you're explaining here, but i'll see what i can come up with.

Example of examinate on a mob:

immagine

**Review by Rukofamicom — APPROVED** [review]
That's a lot of \n removals. 
This was kind of a headache to review and I hope I didn't miss something.

**Pull request merged**